docs/pages/guides/using-firebase.mdx

```diff
@@ -46,215 +76,126 @@ const firebaseConfig = {
+/>
 
-We can choose different login methods that make sense to our application. The login method choice is orthogonal to the Firebase RTDB access, however, we do need to let Firebase know how we have setup our login system so that it can correctly assign authentication tokens that match our user accounts for data access control. You can use anything you want, roll your own custom login system, or even forego it altogether if all your users can have unrestricted access - though unrestricted access is strongly discouraged and instead Firebase recommends using their [_Anonymous_ authentication provider](https://firebase.google.com/docs/auth/web/anonymous-auth).
+<BoxLink
+  title="Storage"
+  description="For more information on how to use Storage, see Firebase documentation."
+  href="https://firebase.google.com/docs/storage/web/start"
+/>
 
-### Facebook Login
+<BoxLink
+  title="Firebase Storage example"
+  description="Learn how to use Firebase Storage in an Expo project with our example."
+  href="https://github.com/expo/examples/tree/master/with-firebase-storage-upload"
+/>
 
-{/* TODO: Mention third-party facebook packages */}
+<BoxLink
+  title="Managing API keys for Firebase projects"
+  description="For more information about managing API Key and unique identifiers in a Firebase project."
+  href="https://firebase.google.com/docs/projects/api-keys"
+/>
 
-A common login system many developers opt for is a simple Facebook login that users are already familiar with.
+## Using React Native Firebase
 
-See the Facebook section of our docs for information on how to set this up. This works just as well with Google and [several others](<https://firebase.google.com/docs/reference/android/com/google/firebase/auth/AuthCredential#getProvider()>).
+[React Native Firebase](https://rnfirebase.io/) provides access to native code by wrapping the native SDKs for Android and iOS into a JavaScript API. Each Firebase service is available as a module that can be added as a dependency to your project. For example, the `auth` module provides access to the Firebase Authentication service.
 
-### Tying Sign-In Providers with Firebase
+### When to use React Native Firebase
 
-Once you have added Facebook login to your React Native app, we need to adjust the Firebase console to check for it. Go to [Firebase Console](http://console.firebase.google.com/) >> _Authentication_ >> _Sign-In method_ tab to enable Facebook as a sign-in provider.
+You can consider using React Native Firebase when:
 
-You can add whichever provider makes sense for you, or even add multiple providers.
+- Your app requires access to Firebase services not supported by the Firebase JS SDK, such as [Dynamic Links](https://rnfirebase.io/dynamic-links/usage), [Crashlytics](https://rnfirebase.io/crashlytics/usage), and so on. For more information, see [React Native Firebase documentation](https://rnfirebase.io/faqs-and-tips#why-react-native-firebase-over-firebase-js-sdk).
```

Comment:
```suggestion
- Your app requires access to Firebase services not supported by the Firebase JS SDK, such as [Dynamic Links](https://rnfirebase.io/dynamic-links/usage), [Crashlytics](https://rnfirebase.io/crashlytics/usage), and so on. For more information on the additional capabilities offered by the native SDK's, see [React Native Firebase documentation](https://rnfirebase.io/faqs-and-tips#why-react-native-firebase-over-firebase-js-sdk).
```
Just suggesting some additional clarity on where this link is going, since I think it's really useful specifics for anyone who is worried about FOMO when choosing the JS SDK.